common/scala/src/main/scala/whisk/core/entity/Identity.scala

```diff
@@ -74,6 +74,53 @@ object Identity extends MultipleReadersSingleWriterCache[Identity, DocInfo] with
         })
     }
 
+    /**
+      * Many subjects can be belonged to only one namespace
+      * Retrieves a key for namespace + subject.
+      *
+      */
+    def get(datastore: AuthStore, namespace: EntityName, subject: Subject)(
```

Comment:
@rabbah ,as your suggestion `look up the docs by primary key:subject`, it seems that doesn't work,
if want to query data from couchdb database, first should create a design document (for example "_design/XXXX"),then create a view.

please refer to https://stackoverflow.com/questions/28945405/how-to-retrieve-all-the-documents-in-couchdb-by-given-key

As you know, openwhiks's couchdb has already to support it, but its query key only support `[auth key]` and `[namespace]`, regarding my case here, i think it is better to query results by `namesapce`, then match the results by `subject` which is passed from http header.